docker-compose.yml

```diff
@@ -61,10 +61,24 @@ services:
     networks:
       - smart-health-links-portal-network
 
+  hapi-fhir:
+    image: jembi/hapi:v7.0.3-wget
+    ports:
+      - 8081:8080
```

Comment:
Suggestion: lets make this port 3447, this align with the Platform port
(personal comment for consistency)